zerver/tests/test_templates.py

```diff
@@ -197,3 +197,14 @@ def test_custom_tos_template(self):
         self.assert_in_success_response([u"Thanks for using our products and services (\"Services\"). ",
                                          u"By using our Services, you are agreeing to these terms"],
                                         response)
+
+    def test_help_guides(self):
+        root = '/home/vagrant/zulip/templates/zerver/help'
```

Comment:
this should be constructed as a relative path, since many developers don't use Vagrant.